docs/source/index.rst

```diff
@@ -189,6 +189,21 @@ The following information is recorded in the models file:
     references for the model. :samp:`x-backrefs` is not a public interface and
     should not be relied upon as it is subject to change.
 
+It is worth noting that the auto-generated models cannot be imported directly.
+The generated models are added to the `open_alchemy.models` package by the
+`init_*` functions.
+
+.. code:: python
```

Comment:
In most of the documentation I have used `.. code-block::` to enable the `:linenos:` directive. An example is here: [https://github.com/jdkandersson/OpenAlchemy/blob/master/docs/source/technical_details/column_modifiers.rst](https://github.com/jdkandersson/OpenAlchemy/blob/master/docs/source/technical_details/column_modifiers.rst) line 33.

Reply:
Addressed ✅